docs/API.md

```diff
@@ -935,7 +936,7 @@ if err != nil {
 }
 ```
 <a name="RemoveObjects"></a>
-### RemoveObjects(bucketName string, objectsCh chan string) errorCh chan minio.RemoveObjectError
+### RemoveObjects(bucketName string, objectsCh chan string) errorCh chan RemoveObjectError
```

Comment:
`minio.` is missed in `RemoveObjectError` and supposes to be `<-chan minio.RemoveObjectError` only